crates/kernel/src/tasks/vm_host.rs

```diff
@@ -73,10 +75,20 @@ pub struct VmHost {
     scheduler_control_sender: Sender<(TaskId, SchedulerControlMsg)>,
     running: bool,
 
-    unsend: PhantomUnsend,
     unsync: PhantomUnsync,
 }
 
+impl Debug for VmHost {
```

Comment:
I've done something similar (and then removed it after debugging) for `MootState`. Opinion on pulling in `derivative` for its `#[derivative(Debug="ignore")]`?

Reply:
Open to the idea as long as the dependencies for it are minimal.